src/Coupon/WCCouponAdapter.php

```diff
@@ -184,7 +184,7 @@ protected function get_wc_coupon_effective_dates( WC_Coupon $wc_coupon ): ?strin
 			$end_date = clone $start_date;
 			$end_date->add( new DateInterval( 'PT1S' ) );
 		}
-		return sprintf( '%s/%s', (string) $start_date, (string) $end_date );
+		return sprintf( '{"startTime": "%s", "endTime": "%s"}', (string) $start_date, (string) $end_date );
```

Comment:
Content API has new field to accept promo effective period.
 this change changes the string format, but does not return the data and put it into the new field?

Reply:
Thanks for your comment. Have updated this part to put the data into the time period field.